client/src/components/Menu/Menu.scss

```diff
@@ -122,31 +130,171 @@
     box-shadow: inset $border-color -1px 0 0;
     position: relative;
     padding-left: 0;
+    justify-content: flex-end;
+    border-top: 1px solid rgba($border-color, .5);
+
+    @media (min-width: 476px) and (max-width: 767px) {
+      .cms-help {
+        &__link {
+          display: none;
+          padding: 0 6px;
+        }
+
+        &__toggle {
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          width: 100%;
+          box-shadow: inset -1px 0 0 #ced5e1;
+
+          &--show {
+            .cms-help__menu {
+              width: auto;
+            }
+
+            .cms-help__link {
+              display: inline-flex;
+            }
+
+            .cms-help__caret {
+              margin-left: 4px;
+            }
+          }
+        }
+      }
+    }
 
     @include media-breakpoint-down(sm) {
-      display: none;
+      .toggle-collapse,
+      .toggle-expand,
+      .sticky-toggle {
+        display: none;
+      }
     }
 
     a,
     a.toggle-expand {
-      width: 50%;
       padding-top: 14px;
       padding-bottom: 14px;
     }
   }
 
+  .cms-help {
+    &__menu {
+      display: flex;
+      align-items: center;
+      background-color: transparent;
+      padding: 13px 12px;
```

Comment:
If we were doing it relative based on $panel-padding then it would look like this:

```padding: $panel-padding-y - .5385rem $panel-padding-x - 0.6155rem;```

Do you think we should stick to the px values as it's cleaner?

Reply:
Take a look at $spacer-sm, which I think equates to 12px (could also do: $panel-padding-x / 1.67). For the 13px its a pretty unique number so that the entire linkable area comes to a 52px height, might be able to use 1rem though?